students/avalvillar/lesson7/html_render.py

```diff
@@ -0,0 +1,143 @@
+"""
+    Author is Antonio V. Alvillar
+    Self-Paced-Online Python 210 UWPCE
+    January 16th 2019
+"""
+
+'''
+This is the Element class that is subclassed from 'object' and will have many
+subclasses from itselt. The overall goal is to be able to render HTML
+documentation as functions are called for particular tags, attributes, and
+indentations/levels.
+'''
+
+#Top level class of project.
+class Element(object):
+
+    #Class level attributes for an empty tag, default indentations
+    tag = ''
+    indentation = 4
+    attrs = {'':''}
```

Comment:
class level attributes is associated with a class vs an instance - do you think `attrs` are specific to a single class and shared across all instances or something specific to an instance of Element?